public/friendlycode/js/require-config.js

```diff
@@ -1,57 +1,25 @@
 var require = {
-  baseUrl: "js",
+  baseUrl: "friendlycode/js",
   shim: {
     underscore: {
```

Comment:
We're using an underscore-based templating system on the front-end for now. We should definitely tear it out when we switch to our browserify approach